petastorm/arrow_reader_worker.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, worker_id, publish_func, args):
         self._local_cache = args[5]
         self._transform_spec = args[6]
         self._transformed_schema = args[7]
+        self._pyarrow_filters = args[8]
```

Comment:
@selitvin - not super familiar with this part of the codebase but is there any reason here why we're using positional args like this instead of keyword args in the `__init__()` or `**kwargs`?

Reply:
Historical reasons. Feel free to clean up :)

Reply:
Cleaned up!